.circleci/config.yml

```diff
@@ -33,12 +33,12 @@ executors:
   # Available resource classes - https://circleci.com/product/features/resource-classes/
   linux-amd64:
     machine:
-      image: default
-      resource_class: xlarge
+      image: ubuntu-2204:2024.01.1
```

Comment:
This was changed to 'default' recently to deal with circleci wanting to change how we define images.

Reply:
thanks Ross! I only moved because CircleCI kept filing it as out of date. Swapping back.